vsts/vsts.yaml

```diff
@@ -1,467 +1,457 @@
-      DPS_GROUPX509_CERTIFICATE_CHAIN: $(DPS-GROUPX509-CERTIFICATE-CHAIN)
-      DPS_TPM_REGISTRATIONID: $(DPS-TPM-REGISTRATIONID)
-      DPS_TPM_DEVICEID: $(DPS-TPM-DEVICEID)
-      PROVISIONING_CONNECTION_STRING: $(PROVISIONING-CONNECTION-STRING)
-      IOTHUB_DEVICE_CONN_STRING_INVALIDCERT: $(IOTHUB-DEVICE-CONN-STRING-INVALIDCERT)
-      IOTHUB_CONN_STRING_INVALIDCERT: $(IOTHUB-CONN-STRING-INVALIDCERT)
-      DPS_GLOBALDEVICEENDPOINT_INVALIDCERT: $(DPS-GLOBALDEVICEENDPOINT-INVALIDCERT)
-      PROVISIONING_CONNECTION_STRING_INVALIDCERT: $(PROVISIONING-CONNECTION-STRING-INVALIDCERT)
-      FAR_AWAY_IOTHUB_HOSTNAME: $(FAR-AWAY-IOTHUB-HOSTNAME)
-      CUSTOM_ALLOCATION_POLICY_WEBHOOK: $(CUSTOM-ALLOCATION-POLICY-WEBHOOK)
-      IOTHUB_PROXY_SERVER_ADDRESS: 127.0.0.1:3128
-      LA_AAD_TENANT: $(LA-AAD-TENANT)
-      LA_AAD_APP_ID: $(LA-AAD-APP-ID)
-      LA_AAD_APP_CERT_BASE64: $(LA-AAD-APP-CERT-BASE64)
-      LA_WORKSPACE_ID: $(LA-WORKSPACE-ID)
+      - task: CopyFiles@2
+        displayName: 'Copy ETL files to the artifacts folder'
+        inputs:
+          SourceFolder: '$(Build.SourcesDirectory)'
 
-  - task: CopyFiles@2
-    displayName: 'Copy TRX files to the artifacts folder'
-    inputs:
-      SourceFolder: '$(Build.SourcesDirectory)'
+          Contents: '**/*.etl'
 
-      Contents: '**/*.trx'
+          TargetFolder: '$(Build.ArtifactStagingDirectory)'
 
-      TargetFolder: '$(Build.ArtifactStagingDirectory)'
+        condition: always()
 
-    condition: always()
+      - task: PublishBuildArtifacts@1
+        displayName: 'Publish Artifact: testresults'
+        inputs:
+          ArtifactName: testresults_windows
 
-  - task: CopyFiles@2
-    displayName: 'Copy ETL files to the artifacts folder'
-    inputs:
-      SourceFolder: '$(Build.SourcesDirectory)'
+        condition: always()
 
-      Contents: '**/*.etl'
+      - task: PublishTestResults@2
+        displayName: 'Publish Test Results **/*.trx'
+        inputs:
+          testRunner: VSTest
 
-      TargetFolder: '$(Build.ArtifactStagingDirectory)'
+          testResultsFiles: '**/*.trx'
 
-    condition: always()
+          testRunTitle: 'Windows Tests'
 
-  - task: PublishBuildArtifacts@1
-    displayName: 'Publish Artifact: testresults'
-    inputs:
-      ArtifactName: testresults_windows
+          platform: Windows
+
+          configuration: 'Debug UT + Release E2E'
+
+        condition: always()
 
-    condition: always()
+  ### WINDOWS Analyzers ###
+  - phase: Analyzers
+    displayName: Analyzers
+
+    condition: succeeded()
+    queue:
+      name: Hosted VS2017
+      timeoutInMinutes: 60
+    steps:
+      - powershell: ./vsts/manual_checkout.ps1
+        displayName: 'GIT checkout'
+        env:
+          COMMIT_FROM: $(COMMIT_FROM)
+
+      - script: |
+          rem Run dotnet first experience.
+          dotnet new
+
+          rem Start build
+          build.cmd -clean -build -configuration Debug
```

Comment:
View this file with whitespace diff off, otherwise it is impossible.

This is the only intentional meaningful line change, matching the parameter changes in build.ps1.